js/main.js

```diff
@@ -0,0 +1,164 @@
+// List of speaker objects
+
+const speaker1 = {
+  imageSrc:
+    'https://res.cloudinary.com/optprime/image/upload/v1646837183/module1-capstone/ross-brawn_qgmohf.jpg',
+  speakerName: 'Ross Brawn',
```

Comment:
- Well done adding the section "Featured speakers" dynamically with JavaScript. Keep this flow 👏

Reply:
Thanks 🙏